package.json

```diff
@@ -3,79 +3,92 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@date-io/date-fns": "^2.14.0",
-    "@emotion/react": "^11.10.4",
-    "@emotion/styled": "^11.10.4",
-    "@material-ui/core": "^4.12.4",
-    "@mui/icons-material": "^5.10.6",
-    "@mui/material": "^5.10.6",
-    "@mui/x-data-grid": "^5.13.0",
-    "@mui/x-date-pickers": "^5.0.0-alpha.6",
-    "@sentry/react": "^6.16.1",
-    "@sentry/tracing": "^6.16.1",
-    "@testing-library/jest-dom": "^5.11.9",
-    "@testing-library/react": "^11.2.3",
+    "@date-io/date-fns": "^2.16.0",
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@mui/icons-material": "^5.11.0",
+    "@mui/material": "^5.11.6",
+    "@mui/x-data-grid": "^5.17.21",
+    "@mui/x-date-pickers": "^5.0.16",
+    "@sentry/react": "^7.34.0",
+    "@sentry/tracing": "^7.34.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
     "@testing-library/react-hooks": "^8.0.1",
-    "@testing-library/user-event": "^12.6.0",
-    "@types/bootstrap": "^5.0.5",
-    "@types/chroma-js": "^2.1.3",
-    "@types/d3": "^6.3.0",
-    "@types/downloadjs": "^1.4.2",
-    "@types/jest": "^28.1.6",
+    "@testing-library/user-event": "^14.4.3",
+    "@types/bootstrap": "^5.2.6",
+    "@types/chroma-js": "^2.1.4",
+    "@types/d3": "^7.4.0",
+    "@types/downloadjs": "^1.4.3",
+    "@types/jest": "^29.4.0",
     "@types/json2csv": "^5.0.3",
-    "@types/node": "^14.14.25",
-    "@types/plotly.js": "^1.54.7",
-    "@types/react": "^17.0.1",
-    "@types/react-bootstrap": "^0.32.25",
-    "@types/react-datepicker": "^4.1.7",
-    "@types/react-dom": "^17.0.0",
-    "@types/react-resize-detector": "^5.0.0",
-    "@types/react-router": "^5.1.11",
-    "@types/react-router-dom": "^5.1.7",
-    "@types/react-select": "^4.0.15",
-    "@types/react-simple-maps": "^1.0.3",
-    "@types/react-test-renderer": "^17.0.1",
-    "@types/styled-components": "^5.1.7",
+    "@types/node": "^18.11.18",
+    "@types/plotly.js": "^2.12.13",
+    "@types/react": "^18.0.27",
+    "@types/react-bootstrap": "^0.32.32",
+    "@types/react-datepicker": "^4.8.0",
+    "@types/react-resize-detector": "^6.1.0",
+    "@types/react-router": "^5.1.20",
+    "@types/react-router-dom": "^5.3.3",
+    "@types/react-simple-maps": "^3.0.0",
+    "@types/react-test-renderer": "^18.0.0",
+    "@types/recharts": "^1.8.24",
```

Comment:
We removed that in https://github.com/GenSpectrum/cov-spectrum-website/commit/f9c50b81781d316895d05c4bed2ab5b7a5431cb5. Did you bring it back intentionally? I think the types were in the wrong version, and are not necessary anymore since recharts 2.0

Reply:
I didn't do this intentionally. This is just a merge conflict I solved with the best of my knowledge and belief. If they are not needed anymore you can remove them again.